pkgs/tools/networking/phodav/default.nix

```
@@ -2,20 +2,21 @@
 , pkgconfig, libsoup, meson, ninja }:
 
 let
-  version = "2.3";
+  version = "2.4";
 in stdenv.mkDerivation rec {
   pname = "phodav";
   inherit version;
 
   src = fetchurl {
     url = "http://ftp.gnome.org/pub/GNOME/sources/phodav/${version}/${pname}-${version}.tar.xz";
-    sha256 = "0ndy5qva6bq7vhk06jq2d4nr5fp98xsdwypg42vjz91h9ii1xxkf";
+    sha256 = "1hxq8c5qfah3w7mxcyy3yhzdgswplll31a69p5mqdl04bsvw5pbx";
   };
 
   mesonFlags = [
     "-Davahi=disabled"
     "-Dsystemd=disabled"
     "-Dgtk_doc=disabled"
+    "-Dudev=disabled"
   ];
 
   nativeBuildInputs = [ libsoup pkgconfig meson ninja ];
```

The dev and lib parts of this package could be split off, so dependencies (like spice-gtk) wouldn't need to ship the binaries and include files of this derivation.

```suggestion
  outputs = [ "out" "dev" "lib" ];

```

I tested this and it seems to work as expected. I'm fairly new to Nix though, perhaps a more experienced maintainer can comment on whether this is a good idea :)

---

@Valodim applied, thanks!